src/main/java/de/doubleslash/keeptime/view/ViewController.java

```diff
@@ -316,7 +316,8 @@ private void initialize() {
             label.setText(DateFormatter.secondsToHHMMSS(seconds));
          }
 
-         updateProjectColorTimeline();
+         final ColorTimeLine mainColorTimeLine = new ColorTimeLine(canvas, model, controller);

```

Comment:
why recreate the timeline each time? just call update on an existing instance :)